mmv1/third_party/terraform/services/sql/resource_sql_database_instance.go.erb

```diff
@@ -775,11 +775,12 @@ is set to true. Defaults to ZONAL.`,
 			},
 
 			"replica_configuration": {
-				Type:     schema.TypeList,
-				Optional: true,
-				MaxItems: 1,
+				Type:       schema.TypeList,
+				Optional:   true,
+				MaxItems:   1,
 				// Returned from API on all replicas
-				Computed: true,
+				Computed:   true,
+				Sensitive:  true,
```

Comment:
The individual sub fields that specifically contain the encrypted data should be flagged sensitive for this update, not the entire list object

Reply:
Thanks, Nick. Unfortunately this is a limitation of TF for these nested messages. Please see https://github.com/hashicorp/terraform-plugin-sdk/issues/201. So I have to mark the whole message as sensitive.

I've tested the version which only marks the sensitive fields in the nested message. TF failed to hide them